src/views/Home.jsx

```diff
@@ -30,26 +36,41 @@ export function Home({ createNewList, listToken, setListToken }) {
 	};
 
 	return (
-		<div>
+		<div className="container">
 			<div className="Home">
 				{listToken && <Navigate to={'/List'} replace={true} />}
-				<p>Welcome to your smart shopping list!</p>
-				<button onClick={handleClick}>Create New List</button>
+				<p className="welcome">Welcome to your smart shopping list!</p>
```

Comment:
Should we mention the app's name here? EG)
Welcome to HoneyDo,
your smart shopping list!
I like the styling you gave the text. You could leave it as is, or if you'd like practice using MUI, you could change this to something like:
`<Typography variant='h4' sx={{ fontWeight: 'bold', textShadow: 'rgba(0, 0, 0, 0.15) 0 1.5px', my:5 }}>Welcome to HoneyDo, </br>your smart shopping list!</Typography>`
The main benefit of going the MUI Typography route is that is makes sure the fonts/font weight stay the same throughout the app.